modules/videopress/class.videopress-video.php

```diff
@@ -175,8 +175,12 @@ public function __construct( $guid, $maxwidth = 0 ) {
 
 		$data = $this->get_data();
 		if ( is_wp_error( $data ) || empty( $data ) ) {
-			$this->error = $data;
-			return;
+			if ( ! apply_filters( 'jetpack_videopress_use_legacy_player', false ) ) {
```

Comment:
Could you add a docblock, something along the lines of `/** This filter is documented in modules/videopress/class.videopress-player.php */` ?

Thanks!